decidim-core/spec/mailers/notification_mailer_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 module Decidim
   describe NotificationMailer do
-    let(:organization) { create(:organization, name: "O'Connor") }
```

Comment:
Same here

Reply:
I understand this, but here we do not have any assertion to test the organization name is actually displayed like it should. Therefore, i do not see it relevant in this place

Reply:
This line come from #2738. The assertion is on https://github.com/decidim/decidim/blob/484ee5f93215a7c4c6ab06d530b2bb9fcf78bbba/decidim-core/spec/mailers/notification_mailer_spec.rb#L29 

Maybe we can change it and leave it as you have, and add a new context to make it explicit? 

```ruby
context "when the organization name has an apostrophe" do
  let(:organization) { create(:organization, name: "O'Connor") }

  it "includes the organization data" do
    expect(mail.body.encoded).to include("O'Connor")
  end
end
```

Reply:
fixed in [1bdd8a5](https://github.com/decidim/decidim/pull/12681/commits/1bdd8a52c9f15118ffd11cdab45d40ed904cb1cb)